File: ExtStore2/app/model/OrderItem.js
Ext.define('Front.model.OrderItem', {
    extend: 'Ext.data.Model',
    fields: ['Id', 'OrderId', 'ProductId', 'ItemsCount', 'ItemPrice'],

    associations: [{
        type: 'belongsTo',
        model: 'Order',
        primaryKey: 'Id',
        foreignKey: 'OrderId',
        //autoLoad: true,
        associationKey: 'Order',
    }],
});

File: ExtStore2/Models/OrderItem.cs
namespace ExtStore2.Models
{
    public class OrderItem
    {
        public int Id { get; set; }

        public int OrderId { get; set; }
        public int ProductId { get; set; }
        public int ItemsCount { get; set; }
        public int ItemPrice { get; set; }

        //public virtual Order Order { get; set; }
        public virtual Product Product { get; set; }
    }
}